cloud_platform_azure/models/cloud_platform.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2016-2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import re
+import os
+
+from odoo import models, api
+from odoo.addons.cloud_platform.models.cloud_platform import FilestoreKind
+from odoo.addons.cloud_platform.models.cloud_platform import PlatformConfig
+
+
+AZURE_STORE_KIND = FilestoreKind("azure", "remote")
+
+
+class CloudPlatform(models.AbstractModel):
+    _inherit = "cloud.platform"
+
+    @api.model
+    def _filestore_kinds(self):
+        kinds = super(CloudPlatform, self)._filestore_kinds()
+        kinds["azure"] = AZURE_STORE_KIND
+        return kinds
+
+    @api.model
+    def _platform_kinds(self):
+        kinds = super(CloudPlatform, self)._platform_kinds()
```

Comment:
```suggestion
        kinds = super()._platform_kinds()
```

Reply:
It will be backported to all Odoo versions, also with python 2.7. I find it easier to have that so it's consistent between all versions even if with python 3 it's not required anymore